src/microbe_stage/editor/HexPopupMenu.cs

```diff
@@ -163,6 +152,16 @@ public bool OnMoveKeyPressed()
         return false;
     }
 
+    protected override void ResolveNodeReferences()
+    {
+        titleLabel = GetNode<Label>(TitleLabelPath);
+        deleteButton = GetNode<Button>(DeleteButtonPath);
+        moveButton = GetNode<Button>(MoveButtonPath);
+        modifyButton = GetNode<Button>(ModifyButtonPath);
+
+        UpdateModifyButton();
```

Comment:
Logically this line doesn't really belong in `ResolveNodeReferences` is it not possible to override _Ready in this class to have this there?